runner/sql.go

```diff
@@ -336,8 +336,8 @@ func importPanel(
 	}
 
 	tname := quote(panel.tableName, qt.identifier)
-	Logln("Creating temp table " + panel.tableName)
-	createQuery := fmt.Sprintf("CREATE TEMPORARY TABLE %s (%s);",
```

Comment:
Ok nvm we aren't just not importing because this can't change unless we're in cache mode.

This code path happens in DataStation when you call `SELECT * FROM DM_getPanel('blubberty')` when using PostgreSQL and MySQL too. In those cases we need to still only create temp tables.

So this code needs to be split by whether or not cache mode is true.

Reply:
I see